docs/portal.properties-Reference.md

```diff
@@ -134,7 +134,34 @@ oncoprint.hide_vus.default=true|false
 ```
 
 # Gene sets used for gene querying
-To change the gene sets used for gene querying, create a JSON file and add gene sets in line with the format of [gene_lists.ts](https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/components/query/gene_lists.ts). The JSON file should contain a structure equal to the part after `const gene_lists = `. Set the path to this file (e.g. `file:/cbioportal/custom_gene_sets.json`) in the following property and restart Tomcat to apply the update.
+To change the gene sets used for gene querying, create a JSON file and add gene sets, following the format specified in the examples below. Set the path to this file (e.g. `file:/cbioportal/custom_gene_sets.json`) in the following property and restart Tomcat to apply the update.
 ```
 querypage.setsofgenes.location=file:/<path>
 ```
+
+## Example with gene names
+```
+[{
+	"id": "Prostate Cancer: AR Signaling",
+	"genes": ["SOX9", "RAN", "TNK2", "EP300", "PXN", "NCOA2", "AR", "NRIP1", "NCOR1", "NCOR2"]
+}, {
+	"id": "Prostate Cancer: AR and steroid synthesis enzymes",
+	"genes": ["AKR1C3", "AR", "CYB5A", "CYP11A1", "CYP11B1", "CYP11B2", "CYP17A1", "CYP19A1", "CYP21A2", "HSD17B1", "HSD17B10", "HSD17B11", "HSD17B12", "HSD17B13", "HSD17B14", "HSD17B2", "HSD17B3", "HSD17B4", "HSD17B6", "HSD17B7", "HSD17B8", "HSD3B1", "HSD3B2", "HSD3B7", "RDH5", "SHBG", "SRD5A1", "SRD5A2", "SRD5A3", "STAR"]
+}]
+```
+
+## Example with specific alterations
+```
+[{
+	"id": "Genes with alterations",
+	"genes": ["TP53: MUT=R273C;", "KRAS: HOMDEL MUT=NONSENSE MUT=NONSTART MUT=NONSTOP MUT=FRAMESHIFT MUT=SPLICE MUT=TRUNC;"]
+}]
+```
+
+## Example with merged gene tracks
+```
+[{
+	"id": "BRCA genes test",
+	"genes": ["[\\\"BRCA genes\\\" BRCA1: MUT=E1258D; BRCA2: HOMDEL MUT=NONSENSE MUT=NONSTART MUT=NONSTOP MUT=FRAMESHIFT MUT=SPLICE MUT=TRUNC;]"]
+}]
+```
```

Comment:
@oplantalech maybe for each example, state what the result will be? E.g. in the first example the result will be 2 sets of genes (so 2 items in the drop down), overwriting the current list, right? Or will these 2 items be appended to the end of the list?

Reply:
@pieterlukasse I've improved the documentation

Reply:
please separate different genes into separate list items. See also https://github.com/cBioPortal/cbioportal/issues/4385. So 
```
[{
	"id": "BRCA genes test",
	"genes": ["[\\\"BRCA genes\\\" BRCA1: MUT=E1258D;", "BRCA2: HOMDEL MUT=NONSENSE MUT=NONSTART MUT=NONSTOP MUT=FRAMESHIFT MUT=SPLICE MUT=TRUNC;]"]
}]
```